support/dynamic_loader/dynamic_library.cpp

```diff
@@ -30,7 +30,7 @@ class InternalDynamicLibrary : public DynamicLibrary {
     lib_with_extension += "-1.dll";
     // We choose RTLD_LAZY because we expect most of the functions
     // in this library to be resolved by other calls to dlsym.
-    lib_ = LoadLibrary(lib_with_extension.c_str());
+    lib_ = LoadLibrary(lib_with_e xtension.c_str());
```

Comment:
This looks wrong.

Reply:
it is suspicious that it compiles with this.

Reply:
I fixed it anyway.